chroot-script.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash -ex
+
+# set up /etc/resolv.conf
+export DEST=$(readlink -m /etc/resolv.conf)
+mkdir -p $(dirname $DEST)
+echo "nameserver 8.8.8.8" > $DEST
+
+# set up hypriot repository
+# and install RPi kernel and firmware
+apt-get update
+wget -q https://packagecloud.io/gpg.key -O - | apt-key add -
+echo 'deb https://packagecloud.io/Hypriot/rpi/debian/ jessie main' > /etc/apt/sources.list.d/hypriot.list
```

Comment:
and probably add the Schatzkiste as well like here https://github.com/hypriot/image-builder-odroid-c1/blob/master/builder/chroot-script.sh#L16-L17 as Hyprio/rpi/debian only contains the kernel, right?